### PR TITLE
completed assignment 1

### PR DESCRIPTION
**Pull request by ShrutiPatelia**
## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — CHANGES_REQUESTED** [review]
A1: Incomplete 

It is valuable to add a detailed PR description, as it can help you in the future when recalling this information as a resource.

Please incorporate more of these details into your assignment, using your own wording:
Q2part(i) Standardizing is important, in particular when using a modeling method that takes into consideration a distance calculation, such as KNN. This will ensure that they are all on a similar scale thus, not having one variable over contributing to the distance calculation
Q2 Part(iv) I would use train_size = 0.75 instead of test_size=0.25. Also, An exercise to try is adding a stratify argument in the train_test_split function, and see if that influences the choice for a best hyperparameter k. 
After completing the above please print the accuracy as your "best k nearest neighbor" value has changed

**Review by FanWang0216 — APPROVED** [review]

**Review by FanWang0216 — APPROVED** [review]
